src/sentry/migrations/0158_create_externalteam_table.py

```diff
@@ -49,8 +50,8 @@ class Migration(migrations.Migration):
                     "provider",
                     sentry.db.models.fields.bounded.BoundedPositiveIntegerField(
                         choices=[
-                            (sentry.models.integration.ExternalProviders(0), "github"),
-                            (sentry.models.integration.ExternalProviders(1), "gitlab"),
+                            (sentry.models.integration.ExternalProviders(200), "github"),
```

Comment:
@wedamija I had to do this for my migration to be able to actually be generated. I don't know if this is okay to do. These tables (`ExternalTeam` and `ExternalUser`) will be dropped eventually, and I wanted the providers to be consistent in how they were numbered.

Reply:
You shouldn't need to modify this migration to have the new migration generated - are you sure it wasn't just that the model hadn't been imported into `models/__init__.py`?